apps/api/src/server/migration/migrators/service-user-migrator.js

```diff
@@ -110,3 +113,12 @@ const mapModelToAddressEntity = ({
 	postcode,
 	country: addressCountry
 });
+
+/**
+ *
+ * @param {import('pins-data-model').Schemas.ServiceUser[]} serviceUserList
+ * @returns {Array<{serviceUsers: import('pins-data-model').Schemas.ServiceUser[], serviceUserType: string}>} - An array of objects, each containing a group of service users and the value of the serviceUserType they were grouped by.
+ */
+const groupServiceUsersByType = (serviceUserList) => {
+	return groupBy(serviceUserList, 'serviceUsers', 'serviceUserType');
```

Comment:
I did take a look at using that, but accessing the value of the key is a little clunky since it produces an object like:
```
{ 'Applicant': [// grouped serviceUsers] }
```
so I'd need to then access the value of the key ('Applicant') in the `sendChunkedEvents` function call. That's why I decided to make a new function that produced the object shape I needed